docker/Dockerfile

```diff
@@ -7,16 +7,19 @@ RUN apt-get update && \
 
 WORKDIR /usr/src/app
 
-COPY docker/package.json .
+# install npm packages
+COPY docker/package.json ./
 RUN npm install
 
-# Bundle app source
-COPY build/ ./build
+# copy our code
 COPY docker/server ./server
-COPY start.sh .
+COPY public ./public
+COPY img ./img
+COPY src ./src
```

Comment:
We shouldn't need to copy `src` over. Let's remove this to be explicit about what is and isn't needed in the Docker env.

Reply:
sgtm